src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -90917,19 +90927,17 @@ INSERT INTO tab_known_transporters VALUES('SLCO1B3','Human','Influx');
 INSERT INTO tab_known_transporters VALUES('SLCO2B1','Human','Influx');
 INSERT INTO tab_known_transporters VALUES('SLCO4C1','Human','Influx');
 INSERT INTO tab_known_transporters VALUES('SLC47A2','Human','Efflux');
-CREATE TABLE tab_known_transporters_containers
-(
-    gene           text,
-    species        text,
-    container_type text,
-    container_name text,
-    transport_type text,
-    membrane       text,
-    constraint tab_known_transporters_containers_pk
-        primary key (gene, species, container_type, container_name),
-    constraint tab_known_transporters_containers_tab_known_transporters_gene_species_fk
-        foreign key (gene, species) references tab_known_transporters (gene, species)
-);
+CREATE TABLE [tab_known_transporters_containers](

+  [gene] text, 

+  [species] text, 

+  [container_type] text, 

+  [container_name] text, 

+  [transport_type] text REFERENCES [tab_active_transport_types]([transport_type]) ON UPDATE CASCADE, 

+  [membrane] text, 

+  CONSTRAINT [tab_known_transporters_containers_pk] PRIMARY KEY([gene], [species], [container_type], [container_name]), 

+  CONSTRAINT [tab_known_transporters_containers_tab_known_transporters_gene_species_fk] FOREIGN KEY([gene], [species]) REFERENCES [tab_known_transporters]([gene], [species]) ON UPDATE CASCADE, 

+  FOREIGN KEY([container_type], [container_name]) REFERENCES [tab_container_names]([container_type], [container_name]) ON UPDATE CASCADE, 

+  CHECK(membrane IN ('Apical', 'Basolateral', 'BloodBrainBarrier', 'Tissue')));

```

Comment:
Added membrane as constraint.
Was thinking about defining another new table (membrane_types or so) - but currently this is the only place where the membrane is used, so I kept is as check constraint.

Reply:
yep/ Not use anywhere else

Reply:
In fact, if we agree to remove BloodBrainBarrier and TIssue and use Apical and Basolateral, then we can only keep the two entries here